test/conformance/enqueue/urEnqueueUSMMemcpy.cpp

```diff
@@ -1,2 +1,140 @@
 // Copyright (C) 2023 Intel Corporation
 // SPDX-License-Identifier: MIT
+
+#include <uur/fixtures.h>
+#include <vector>
+
+struct urEnqueueUSMMemcpyTest : uur::urQueueTest {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urQueueTest::SetUp());
+
+        bool device_usm{false};
+        ASSERT_SUCCESS(
+            urDeviceGetInfo(device, UR_DEVICE_INFO_USM_DEVICE_SUPPORT, sizeof(bool), &device_usm, nullptr));
+
+        if (!device_usm) {
+            GTEST_SKIP_("Device USM is not supported");
+        }
+
+        ur_usm_mem_flags_t flags;
```

Comment:
```suggestion
        ur_usm_mem_flags_t flags{};
```
I think you should zero the flags - there is no guarantee that it will be zero and may result in some of the flags being set.